recipes/panic/meta.yaml

```diff
@@ -0,0 +1,104 @@
+{% set name = "panic" %}
+{% set version = "9.4.2" %}
+{% set python_min = "3.9" %}
+
+package:
+  name: panic-suite
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/panic-{{ version }}.tar.gz
+  sha256: f2772142ac60f69a8927a98317213804abb5becff565a852438288f97353827b
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  # hint from cibuildwheel
+  script_env:
+    - MACOSX_DEPLOYMENT_TARGET=10.9
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ stdlib("c") }}
```

Comment:
This package is pure python - you don't need any compiler or stdlib("c") to build it
Probably why you have this error about c_stdlib_version and MACOSX_DEPLOYMENT_TARGET

Reply:
Hi @benjee, I added those requirements and test due to conda-forge linter comments